grails-app/services/de/iteratec/osm/system/MissingJobResultCheckService.groovy

```diff
@@ -0,0 +1,103 @@
+package de.iteratec.osm.system
+
+import de.iteratec.osm.measurement.environment.wptserver.JobResultPersisterService
+import de.iteratec.osm.measurement.schedule.Job
+import de.iteratec.osm.result.JobResult
+import de.iteratec.osm.result.JobResultStatus
+import grails.gorm.transactions.Transactional
+import org.joda.time.DateTime
+import org.quartz.CronExpression
+
+@Transactional
+class MissingJobResultCheckService {
+    static final int FIRST_CHECK_DAYS_AGO = 1
+    static final long SCHEDULE_THRESHOLD_MINUTES = 3
```

Comment:
You can simply use an `int` here, so there is no need to use `.toInteger()` when used below.